src/blocklypropclient.js

```diff
@@ -497,27 +449,21 @@ function establish_socket() {
                     $('#compile-console').val('');
 
                 } else if (ws_msg.action === 'message-compile') {
-                    if (client_version >= minCodedVer) {
-                        //Messages are coded; check codes, log all and filter out nsDownloading duplicates
-                        var msg = ws_msg.msg.split("-");
-                        if (msg[0] != nsDownloading || !launcher_download) {
-                            launcher_result = launcher_result + msg[1] + "\n";
-                            launcher_download |= (msg[0] == nsDownloading);
-                        }
-                        if (msg[0] == nsDownloadSuccessful) {
-                            //Success! Keep it simple
-                            $('#compile-console').val($('#compile-console').val() + ' Succeeded.');
-                        } else if (msg[0] == neDownloadFailed) {
-                            //Failed! Show the details
-                            $('#compile-console').val($('#compile-console').val() + ' Failed!\n\n-------- loader messages --------\n' + launcher_result);
-                        } else {
-                            //Show progress during downloading
-                            $('#compile-console').val($('#compile-console').val() + ".");
-                        }
+                    //Messages are coded; check codes, log all and filter out nsDownloading duplicates
```

Comment:
Checked with @PropGit - all BPL loading messages are coded, so this test is unnecessary.